### PR TITLE
mgr: install tox deps from wheelhouse

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/tox.ini [hunk 1]
```diff
@@ -27,6 +27,8 @@ setenv =
     WEBTEST_INTERACTIVE = false
 commands =
     pytest {posargs}
+install_command =
+    pip install --find-links {toxinidir}/wheelhouse {opts} {packages}
```
Comment (epuertat) [inline src/pybind/mgr/dashboard/tox.ini]: Doesn't `export PIP_FIND_LINKS="$TOX_INI_DIR/wheelhouse"` work? [[1]](https://pip.pypa.io/en/stable/user_guide/#environment-variables)

Comment (tchaikov) [reply src/pybind/mgr/dashboard/tox.ini]: @epuertat do you prefer export `PIP_FIND_LINKS` in run_tox.sh  ?

Comment (epuertat) [reply src/pybind/mgr/dashboard/tox.ini]: That way you don't need to redefine install_command in any tox file. But I don't have any strong objection, so your call.

Comment (tchaikov) [reply src/pybind/mgr/dashboard/tox.ini]: @epuertat makes sense to me. updated and repushed.

**Pull request merged**